utils/update-consul-conf

```diff
@@ -307,6 +314,26 @@ append_hax_svc() {
             \"status\": \"warning\"
           }
         ]
+    },
+    {
+      \"id\": \"$http_id\",
+      \"name\": \"hax_http\",
+      \"address\": \"$addr\",
+      \"port\": $http_port,
```

Comment:
No, this port is used to generate hax motr endpoint, instead. add http port to the `meta` parameters, see the `transport_type`.

Reply:
Moved `http_port` to the `hax` service metadata.